downstream/modules/eda/con-event-streams.adoc

```diff
@@ -0,0 +1,39 @@
+
+[id="event-streams"]
+
+= Event streams
+
+[role="_abstract"]
+Event streams can send events from remote systems to {EDAcontroller}. In a typical set-up, a server sends data to an event stream over the internet to an {EDAName} event stream receiver. When the data comes over the internet, the request must be authenticated. Depending on the webhook vendor or remote system, the authentication method could differ.
+
+{EDAcontroller} supports 7 different event stream types.
+ 
+.Event Stream Types
+[cols="a,a,a"]
+|===
+| Type | Description | Vendors
+
+h| HMAC | Hashed Message Authentication Code (HMAC). Uses a shared secret between {EDAcontroller} and the vendors webhook server. This guarantees message integrity. | Github
+
+h| Basic Authentication | Uses HTTP basic authentication. | Datadog, Dynatrace
+
+h| Token Authentication | Uses Token Authentication. Usually the HTTP Header is *Authorization* but some vendors like Gitlab use *X-Gitlab-Token*. | Gitlab, ServiceNow
+
+h| OAuth2 | Uses Machine-to-Machine (M2M) mode with a grant type called *client credentials*. The token is opaque. | Dynatrace
+
+h| OAuth2 with JWT | Uses M2M mode with a grant type called *client credentials*. The token is JSON Web Token (JWT). | Datadog
+
+h| ECDSA | Elliptic Curve Digital Signature Algorithm | SendGrid, Twilio
+
+h| Mutual TLS | Needs the vendor's CA certificate to be present in our servers at startup. This supports non-repudiation. 
+| PagerDuty
+|===
+
+{EDAcontroller} also supports 4 other specialized event streams that are based on the 7 basic event stream types: 
```

Comment:
seven

Reply:
Same as my comment above, as I am consistently using numbers. As long as the approach is consistent, it is acceptable.